pkg/asset/agent/image/ignition.go

```diff
@@ -170,6 +172,13 @@ func (a *Ignition) Generate(dependencies asset.Parents) error {
 		return err
 	}
 
+	// Use bootstrap script to get container images
+	err = bootstrap.AddStorageFiles(&config, "/usr/local/bin/release-image.sh",
+		"bootstrap/files/usr/local/bin/release-image.sh.template", agentTemplateData)
```

Comment:
Instead of `agentTemplateData`, we could just pass `struct {ReleaseImage string}{ReleaseImage: agentManifests.ClusterImageSet.Spec.ReleaseImage}` to avoid any inadvertent cross-pollination from the other variables.

Reply:
Good idea, changed